homeassistant/components/twinkly/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Config flow to configure the Twinkly integration."""
+
+import asyncio
+import logging
+
+from aiohttp import ClientError
+import twinkly_client
+from voluptuous import Required, Schema
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+
+from .const import (
+    CONF_ENTRY_HOST,
+    CONF_ENTRY_ID,
+    CONF_ENTRY_MODEL,
+    CONF_ENTRY_NAME,
+    DEV_ID,
+    DEV_MODEL,
+    DEV_NAME,
+)
+
+# https://github.com/PyCQA/pylint/issues/3202
+from .const import DOMAIN  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TwinklyConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle twinkly config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle config steps."""
+        host = user_input[CONF_HOST] if user_input else None
+
+        schema = {Required(CONF_HOST, default=host): str}
+        errors = {}
+
+        if host is not None:
```

Comment:
You can reverse this condition in order to save some indention.

Reply:
If I revert the condition, it would mean that I've to repeat the `return self.async_show_form...`  for the error handling (invalid host). Usually I prefer to indentation over repetition 😋, so unless you tell me that it's the common pattern in python, I would prefer to keep it as is.

Reply:
fair enough